server/pxf-jdbc/src/main/java/org/greenplum/pxf/plugins/jdbc/JdbcResolver.java

```diff
@@ -141,6 +128,11 @@ public class JdbcResolver extends JdbcBasePlugin implements Resolver {
 
     private static final Logger LOG = LoggerFactory.getLogger(JdbcResolver.class);
 
+    private static final String dateWideRangeWarnMsg = "Failed to use the standard formatter, so we had to fallback to the wide date range formatter. " +
+            "However, there is a performance penalty. To have better performance, specify date_wide_range=true at the table definition";

```

Comment:
```suggestion
            "However, there is a performance penalty. To have better performance, specify date_wide_range=true in the table definition";

```

they should also be able to set the value in the jdbc-site.yml (but this would affect all tables that use this profile)